packages/@vuepress/test-utils/package.json

```diff
@@ -33,6 +33,10 @@
     "vue-jest": "^4.0.0-beta.1",
     "vue-template-compiler": "^2.5.16"
   },
+  "peerDependencies": {
```

Comment:
Why now would we need these two dependencies as peerDependencies?

Reply:
in test utils we no mention of vuejs. It's just a protection to say i need at least vue 2x to work. It installing nothing